packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -285,6 +302,20 @@ export class WorkspaceCommandContribution implements CommandContribution {
         }
         return parentUri.resolve(base);
     }
+
+    private async openWorkspaceRootConfig(): Promise<void> {
+        const rootConfigUri = await this.workspaceService.getWorkspaceConfigFileUri();
+        if (!rootConfigUri) {
+            return;
+        }
+        const path = rootConfigUri.path.toString();
+        if (!(await this.fileSystem.exists(path))) {
```

Comment:
we should encapsulate it in `WorkspaceService`, clients should not know about the config file format